clang/test/CodeGenCXX/linetable-eh.cpp

```diff
@@ -33,12 +33,12 @@ class Foo
     bool bar();
     virtual shared_ptr<Context> GetContext () = 0;
 };
-# 253 "Foo.cpp" 3
+# 253 "Foo.cpp"
```

Comment:
Can you explain what this change is for? Should it have INTEL markers?

Reply:
The "3" is a pre-processor annotation that indicates this is a system file.  I believe whether this file is a system file or not is not relevant to the testcase, but after this PR debug info is generated differently.  Thus, I removed the system file annotation.

In my analysis of CMPLRLLVM-24314 there are many ways to reduce additional debug info.  A large amount of debug info was for declarations in system headers, which I suspect is not useful for many debugging situations.  This will not help reduce debug information for user data structures.   Separate sub-JIRAs CMPLRLLVM-53145 will track reducing debug info in sycl-post-link.